box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequest.java

```diff
@@ -334,6 +337,9 @@ public String getStringBody() throws UnsupportedEncodingException {
                 }
                 mStringBody = createQuery(stringMap);
                 break;
+            case JSON_PATCH:
```

Comment:
Since this is just JSON parsing, let's move it as part of the JSON block (as an if statement) and make JSON_PATCH fall through to the same logic.